pages/a/1001-node-crate-cluster-with-microsoft-azure.html

```diff
@@ -0,0 +1,185 @@
+title: A 1001 Node Crate Cluster with Microsoft Azure
+author: Chris Ward
+description: We show how to scale your Crate cluster for billions of records and beyond as simple as possible by launching a 1001 node cluster on Microsoft Azure infrastructure.
+created: 2016-03-30
+status: publish
+post_type: post
+tags: azure, docker, cloud
+category: developernews
+
+[Crate](https://crate.io/) is striving to be the most scalable database in our industry. We aim to make scaling your data from millions to billions of records and beyond as simple as possible.
```

Comment:
"striving to be..." we focus on simplicity, horizontal scaling. probably "the easiest to scale"? please reword